packages/account/src/Components/trading-assessment/trading-assessment-form.jsx

```diff
@@ -54,6 +61,10 @@ const TradingAssessmentForm = ({
                 setCurrentQuestionDetails(prev_state_question => {
                     const next_state_question_index = prev_state_question.current_question_index + 1;
                     localStorage.setItem('current_question_index', next_state_question_index);
+                    // Sub section form progress is optional
+                    if (typeof setSubSectionIndex === 'function') {
+                        setSubSectionIndex(next_state_question_index);
+                    }
```

Comment:
would it be better if we pass a flag to differentite a trading assessment new user form and existing user form instead of checking the type